### ISSUE TITLE
Patcher Request - Swap Static crates/barrels/boxes/etc. with lootable variants.

### ISSUE THREAD

**Issue by mrudat** [Synthesis-Collective/Development#6]

**Comment by glyphy** [comment]
so it would be something similar to https://www.nexusmods.com/skyrimspecialedition/mods/19521 I'm all for it.

**Comment by theSkyseS** [comment]
working on it

**Comment by tr4wzified** [comment]
Did you see this, @theSkyseS? https://github.com/Synthesis-Collective/MakeFirewoodPilesIntoContainers/

Dunno how advanced you want to make this, but the above patcher is a start :)

**Comment by theSkyseS** [comment]
Yes, I seen this. At first I wanted to PR in this patcher, but this is patcher for wizkid's piles. I want to generate containers in code, to remove dependece on plugin.

**Comment by tr4wzified** [comment]
Ah, good call!

**Comment by theSkyseS** [comment]
I can say it's partly done
https://github.com/theSkyseS/LootableCrates
There is also this:
https://github.com/theSkyseS/HarvestThoseMammothSkulls for mammoth skulls
https://github.com/Synthesis-Collective/MakeFirewoodPilesIntoContainers for firewood piles
and also this:
https://github.com/m-vlad/LootableThings-TrainingDummies-Patcher
and this:
https://www.nexusmods.com/skyrimspecialedition/mods/49250

**Comment by theSkyseS** [comment]
I can probably add more things to my patcher, but there is stuff like mammoth skulls, hanging taproots, etc. that requires meshes editing and I don't really know how to do it dynamically.
